vendor/scripts.js

```diff
@@ -0,0 +1,29 @@
+$(function(){
+	$('button').click(function(){
+		 var requestURL = 'http://api.census.gov/data/2013/acs5?get=NAME,B01001_001E&for=school+district+(elementary):*&in=state:50&key=ff063179a36395ac91e4f4acf79b25f1678fc074';
+	    $.ajax({
+	        type: 'GET',
+	        url: requestURL,
+	        dataType: 'json',
+	        success: Automatic,
+	        error: function (data) {
+	            console.log("ERROR:  " + data);
+	        }
+	    });
+
+	    function Automatic ( data ) {
```

Comment:
How would you describe what this function is doing? A more descriptive name would help. Also, I'd move it above line 2, so that a) it isn't relying on [hoisting](http://www.adequatelygood.com/JavaScript-Scoping-and-Hoisting.html) to be available as the `success` callback above, and b) it doesn't have to be re-declared on every button click.